site/content/en/docs/Guides/Matchmaker/_index.md

```diff
@@ -1,77 +1,83 @@
 ---
-title: "Guide to Matchmaking using Open Match"
+title: "Matchmaking using Open Match"
 linkTitle: "Matchmaking using Open Match"
 weight: 1
 description: >
   A deep dive into building an Open Match based Matchmaker.
 ---
 
-This section is work in progress. Please revisit later.
+## Objective
 
-## Objectives
+The guide presents a high level Online Game Services architecture that leverages Open Match for matchmaking. It expains in details the core customizations that need to be added to Open Match to build your Matchmaker. It also takes a deep dive into the external components that interact with Open Match. Along the way, it covers the Open Match concepts needed for building these components.
 
-This guide explains the core Open Match concepts you need to understand to build your Open Match based Matchmaker. The guide presents the key components of an Online Game Services architecture, their responsibilities and their interactions with core Open Match Services. The guide also explains in details, the customizations to be added to Open Match.
+This guide is a pre-reading for the tutorial for [building a basic Matchmaker]({{< relref "../../Tutorials/Matchmaker101/_index.md" >}}). You can use this [Tutorial Solution](https://github.com/googleforgames/open-match/blob/{{< param release_branch >}}/tutorials/matchmaker101) as a reference when going through each component in this guide.
 
 ## Game Services Architecure
 
-Here is a high level overview of an online Game Services architecture with details of which Open Match Core services do the different components interact with:
+![Architecture](../../../images/architecture.png)
 
-![Life of a Match](../../../images/architecture_omdetails.jpg)
+### Open Match Core
 
-### Open Match Services
+Open Match comprises of a set of services hosted in a Kubernetes cluster. The core functionality of Open Match is exposed by these services over gRPC and HTTP.
```

Comment:
nit: These services expose the core functionality of Open Match over gRPC and HTTP